cloud_run/shelf_server/.dockerignore

```diff
@@ -0,0 +1,9 @@
+.dockerignore
```

Comment:
The folder name we use for samples uses the Google service name / short name. In this case it would be `run/` instead of `cloud_run/`.